aspen/go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/synnaxlabs/aspen
 
-go 1.22
+go 1.22.7
```

Comment:
If we're going to upgrade deps here, we might as well upgrade to go 1.23 as well across all of our packages.